Assets/HoloToolkit/UI/Scripts/UsabilityUtilities.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// A helper class for making applications more usable across different devices.
+    /// </summary>
+    public static class UsabilityUtilities
```

Comment:
This is a great idea!  I'm concerned about all of the hard coded values.  A thought would be to rework this such that the api is float GetUsabilityScaleFactor(Camera camera, float targetPPD) and return the scale factor required to get the targetPPD.  This way you don't need to have any of the hard coded values (aside from the min/max scale factor) and I think you'll get a good result.